src/main/java/com/example/recipebook/model/Recipe.java

```diff
@@ -0,0 +1,74 @@
+package com.example.recipebook.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+public class Recipe {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String name;
+    private String description;
+    private int likes;
+    private String url;
+
+
+    @ManyToOne(cascade = CascadeType.PERSIST)
+    private Category category;
+
+    public Long getId() {
```

Comment:
Wcześniej generowałes rzeczy lombokiem a tutaj robisz to ręcznie - trzymaj się jednego stylu.